src/test/java/tdd/setup/CalculatorTest.java

```diff
@@ -29,4 +29,16 @@ void calculatorCanDoFiveMultiplyingThree() {
         calc.pressEquals();
         assertEquals("-15", calc.readScreen());
     }
+
+    //test case two
+    @Test
+    @DisplayName("should return a result for operations on digits greater 9")
+    void calculatorOperationsOnDigitsGreaterNine() {
+        Calculator calc = new Calculator();
+        calc.pressDigitKey(16);
+        calc.pressOperationKey("x");
+        calc.pressDigitKey(16);
+        calc.pressEquals();
+        assertEquals("256", calc.readScreen());
```

Comment:
Das ist eigentlich etwas unsauber, da der Online-Taschenrechner nur Eingaben von Ziffern erlaubt. Für die Zahl 16 muss also die Ziffer 1 und 6 eingeben werden, was gleichbedeutend mit zwei Aufrufen von `pressDigitKey()` ist

```suggestion
        Calculator calc = new Calculator();
        calc.pressDigitKey(1);
        calc.pressDigitKey(6);
        calc.pressOperationKey("x");
        calc.pressDigitKey(1);
        calc.pressDigitKey(6);
        calc.pressEquals();
        assertEquals("256", calc.readScreen());
```